src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/TierCalculationTest.kt

```diff
@@ -1,204 +1,218 @@
 package uk.gov.justice.digital.hmpps.hmppstier.integration
 
-import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS
 import org.mockserver.model.HttpRequest.request
 import org.mockserver.model.HttpResponse.response
 import org.mockserver.model.MediaType.APPLICATION_JSON
 import org.springframework.beans.factory.annotation.Autowired
-import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ChangeLevel
-import uk.gov.justice.digital.hmpps.hmppstier.domain.enums.ProtectLevel
-import uk.gov.justice.digital.hmpps.hmppstier.service.TierCalculationService
+import uk.gov.justice.digital.hmpps.hmppstier.service.TierCalculationRequiredEventListener
 
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestInstance(PER_CLASS)
 class TierCalculationTest : MockedEndpointsTestBase() {
 
   @Autowired
-  lateinit var service: TierCalculationService
+  lateinit var listener: TierCalculationRequiredEventListener
```

Comment:
Now more of an outside-in test where we call the listener and check that the update happens as expected. This makes it a bit easier to refactor the service - changing the return types for example made a real mess of this

If/when we get requirements for the controller, this kind of test would be easier to write against that.